File: src/test/java/no/nav/sosialhjelp/soknad/web/rest/feil/ThrowableMapperTest.java
//package no.nav.sosialhjelp.soknad.web.rest.feil;
//
//import no.nav.sosialhjelp.soknad.business.exceptions.SamtidigOppdateringException;
//import org.junit.jupiter.api.Test;
//import org.junit.jupiter.api.extension.ExtendWith;
//import org.mockito.junit.jupiter.MockitoExtension;
//
//import javax.ws.rs.core.Response;
//
//import static org.assertj.core.api.Assertions.assertThat;
//
//@ExtendWith(MockitoExtension.class)
//class ThrowableMapperTest {
//
//    final private ThrowableMapper mapper = new ThrowableMapper();
//
//    @Test
//    void skalGi409ConflictVedSamtidigOppdateringException() {
//        Response response = mapper.toResponse(new SamtidigOppdateringException("Mulig versjonskonflikt..."));
//        assertThat(response.getStatus()).isEqualTo(409);
//    }
//}

File: src/test/kotlin/no/nav/sosialhjelp/soknad/common/rest/feil/ThrowableMapperTest.kt
package no.nav.sosialhjelp.soknad.common.rest.feil

import no.nav.sosialhjelp.soknad.common.exceptions.SamtidigOppdateringException
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

internal class ThrowableMapperTest {

    private val mapper = ThrowableMapper("loginserviceUrl")

    @Test
    fun skalGi409ConflictVedSamtidigOppdateringException() {
        val response = mapper.toResponse(SamtidigOppdateringException("Mulig versjonskonflikt..."))
        assertThat(response.status).isEqualTo(409)
    }
}
